pylint_odoo/test_repo/eleven_module/__init__.py

```diff
@@ -1,2 +1,3 @@
 from . import models
 from . import utf8_models
+from .tests import TestModel
```

Comment:
I think you are missing the python file name test_model1.py instead of the classname here

```suggestion
from .tests import test_model1
```

(maybe that is the reason it is detected a "external dependency"

Reply:
You right, done.